### PR TITLE
updated basecard, added bgs and styling

### PR DESCRIPTION
**Pull request by freedannyv**

### CODE REVIEW

File: src/views/GameScores.vue [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
         >
           <!-- PLAYER NAMES -->
-          <p class="text-005D63 font-kalam text-xl mr-16">{{ player.name }}</p>
+          <p class="text-005D63 mr-16 font-kalam">{{ player.name }}</p>
           <!-- SCORE INPUT -->
           <input
             type="number"
-            class="h-8 w-8 rounded-full border-aeb49a border text-3b9d11 focus:outline-none text-center"
+            class="h-10 w-10 rounded-full border-aeb49a border text-3ac792 focus:outline-none text-center flex items-center justify-center"
           />
```
Comment (giteshk11) [inline src/views/GameScores.vue:29]: I think its better if restrict the score to double digits
Right now user input N digits number

File: src/components/utilities/BaseCard.vue [hunk 2]
```diff
@@ -1,38 +1,15 @@
...
-  }
-};
+
+}
 </script>
 
-<style></style>
+<style>
+
+</style>
```
Comment (giteshk11) [inline src/components/utilities/BaseCard.vue:15]: This whole component just has 3 lines of code, which could be just written directly without using this component.
Please find an alternative way

File: src/styles/tailwind.css [hunk 3]
```diff
@@ -11,7 +11,7 @@
 @import './animations.css';
 
 .confirm {
-  @apply py-2 px-8 uppercase rounded-full shadow-md bg-ff6350 text-base;
+  @apply py-2 px-8 uppercase rounded-full shadow-lg bg-ff6350 text-base;
```
Comment (giteshk11) [inline src/styles/tailwind.css:14]: You can place this class in the component's CSS style

File: src/styles/tailwind.css [hunk 4]
```diff
@@ -11,7 +11,7 @@
...
 }
 
 .back {
```
Comment (giteshk11) [inline src/styles/tailwind.css:17]: Same for this

**Review by giteshk11 — CHANGES_REQUESTED** [review]
Left some feedback comments

File: src/views/GameScores.vue (hunk 1 above)
Comment (freedannyv) [reply src/views/GameScores.vue:29]: I'm not entirely sure on how to do this

File: src/components/utilities/BaseCard.vue (hunk 2 above)
Comment (freedannyv) [reply src/components/utilities/BaseCard.vue:15]: I made a class in the tailwind.css styles named .card

File: src/styles/tailwind.css (hunk 3 above)
Comment (freedannyv) [reply src/styles/tailwind.css:14]: corrected

**Pull request merged**